thesis-de/content-de.tex

```diff
@@ -1892,45 +1892,46 @@ \subsection{SSD Userstory}
-am häufigsten ausfallen kann.
+dies für einen schnelleren Datenzugriff. SSDs haben jedoch, wie auch HDDs einen
+Lebenszyklus, der bei SSDs schneller erreicht werden kann, als bei 
+handelsüblichen HDDs. Der Administrator soll hier darüber Benachrichtigt werden,
+sobald von einer SSD ein kritischer Schreib- oder Lesezyklus erreicht worden
+ist, um die SSD, bereits vor dem Ausfall austauschen zu können. Ebenfalls soll
+hier auch eine Statistik ausgegeben werden, welche SSD von welchem Hersteller in
+der Wahrscheinlichkeit am häufigsten ausfallen kann.
 \\textbf{Akzeptanzkriterien:}\\
 
 \begin{outline}
-  \1 Die Ausgabe und Visualisierung in einen Diagramm, ob Balken- oder
-  Graphendiagramm, soll jeweils für jede einzelne Instanz angezeigt werden
+  \1 Die Ausgabe und Visualisierung in einem Diagramm, welches vorzugsweise in
+  einem Balken- oder Graphendiagramm für jede einzelne Instanz ausgegeben
+  werden kann
   \1 Auflistung einer Statistik in Form eines Balkendiagramms, über die
   Hersteller, welche am wahrscheinlich häufigsten in den Instanzen ausfallen
   \1 Eine Benachrichtigung, ob per Email oder andere Kommunikationswege,
-  wenn ein kritischer Schreib- oder Lesezyklus erreicht worden ist.
+  wenn ein kritischer Schreib- oder Lesezyklus erreicht worden ist
 \end{outline}
 \mr%
 
 \subsection{CPU Userstory}
-\\textbf{Name der Story:}\\ Auswertung der CPU Auslastung von jeder Instanz, um Engpässe zu
+\\textbf{Name der Story:}\\ Auswertung der CPU Auslastung jeder Instanz, um Engpässe zu
 erkennen.
-\\textbf{Beschreibung:}\\ Die CPUs der heutigen Zeit werden immer schneller und besser, von
-der Taktrate, bis hin zur Verarbeitungsgeschwindigkeit der Prozesse. Bei 
-Virtuellen Maschine, werden die CPUs jeweils pro Virtueller Maschine zugewiesen,
-so kann eine Maschine 2 CPU Kerne mit z.B. 2,5 GHz Takrate zugewiesen werden.
-Auf jeder Virtuellen Maschine laufen andere Dienste, weshalb eine VM
-unterschiedliche CPUs zugewiesen haben. Manchmal kommt es hier vor, das eine VM
-zu viel Rechenleistung, bedingt durch ausgeführte Dienste benötigt. Wenn eine
-VM mehr Rechenleistung benötigt, als zugewiesen ist, kann es hier zu
-Verzögerungen kommen. Um dies zu erkennen, benötigt der Administrator, eine
-detailierte Übersicht in Form eines Graphen, wieviel CPU Leistung von einer
-Virtuellen Instanz verwendet wird, um hier einen Engpass bereits vorab zu
-erkennen und der VM mehr CPU Leistung zur Verfügung stellen zu können.
+\\textbf{Beschreibung:}\\ Die CPUs der heutigen Zeit werden in der Taktrate und
+Verarbeitungsgeschwindigkeit der Prozesse immer schneller und besser. Bei
+virtuellen Maschine, werden die CPUs jeweils pro Instanz zugeteilt, so kann eine
+VM 2 CPU Kerne mit z.B. 2,5 GHz Takrate zugewiesen haben. Auf jeder virtuellen
+Maschine laufen andere Dienste, weshalb jede VM unterschiedliche CPUs zugeteilt
+haben kann. Hier kann es vorkommen, dass eine VM mehr CPU Leistung benötigt, als
```

Comment:
Das ist nicht ganz richtig. 
Eine VM hat wirklich nur das was Ihr zugewiesen ist, also ein virtueller Kern. 
Wenn der Kunde nun exterme mathematische Berechnungen durchführt, hat er eben nur die Leistung von einem virtuellen Kern -> muss sich mehr dazu kaufen.

Was du hier meinst ist das der physiche CPU Kern voll ist und auch alle anderen physichen Kerne (Swap nicht mehr möglich). Das kommt aber nur wenn wirklich alle VM's gleichzeitigt arbeiten. 

Trenn das in dem Text besser, sonst wird das eine mit dem anderen verwechselt.